index.html

```diff
@@ -16,30 +16,205 @@
+        </section>
+
+        <section class="about-container">
+            <div class="about">
+                <h3>About Myself</h3>
+                <p class="text">Hello I’m a software developer! I can help you build a product , feature 
+                    or website. Look through some of my work and experience! If you like what 
+                    you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                <label class="connect">LET'S CONNECT</label>
+                <ul class="social-media-icons">
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-linkedin"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-sharp fa-solid fa-m"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-brands fa-github"></i></a>
+                    </li>
+                    <li>
+                        <a href="#" class="icons"><i class="fa-solid fa-hand-peace"></i></a>
+                    </li>
+                </ul>
+                <button class="btn">Get my resume</button>
+
+                <ul class="menu-list">
+                    <ul class="dropdown">
+                        <li class="open">
+                            <label class="dropdown-label">Languages</label>
+                            <i class="fa-solid fa-chevron-down arrow"></i>
+                        </li>
+                        <li class="languages">
+                            <img src="./images/JavaScipt.png" alt="JavaScipt">
+                            <p>JavaScript</p>
+                        </li>
+                        <li class="languages">
+                            <img src="./images/HTML.png" alt="JavaScipt">
+                            <p>HTML</p>
+                        </li>
+                        <li class="languages">
+                            <img src="./images/CSS.png" alt="JavaScipt">
+                            <p>CSS</p>
+                        </li>
+                    </ul>
+                    <ul class="dropdown">
+                        <li class="close">
+                            <label class="dropdown-label">Frameworks</label>
+                            <i class="fa-solid fa-chevron-right arrow"></i>
+                        </li>
+                        <hr>
+                    </ul>
+                    <ul class="dropdown">
+                        <li class="close">
+                            <label class="dropdown-label">Skills</label>
+                            <i class="fa-solid fa-chevron-right arrow"></i>
+                        </li>
+                        <hr>
+                    </ul>
+                </ul>
```

Comment:
- [ ] As requested by the previous reviewer kindly use a nested unordered list to markup your list of technologies. Note that 'languges', 'frameworks', and 'skills' are list items of the parent unordered list and 'html', 'css', and 'javascirpt' are list items under languages. Kindly use the markup below as a guide;
```html

<ul>
  <li>Languages
    <ul>
        <li>JavaScript</li>
        <li>HTML</li>
        <li>CSS</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```